File: C Modules/Assignments/Assignment no 9/Assignment no 7/Question-2/Assignment no 3/Type-3/Question_5.c
// Check the given number is Armstrong number or not..
// Input: n = 153
// Output: Armstrong
#include<stdio.h>
int armstrong(int n[10],int i[10],int j[10]);

int main()
{
    int n[10],i[10],j[10];
   int result=armstrong(n,i,j);
   int *ptr2=&result;
    if (*ptr2==0)
    {
        printf(" is armstrong number");
    }
    else{
        printf(" is not armstrong numb");
    }
    return 0;
    
}

int armstrong(int n[10],int i[10],int j[10])
{
    int sum=0;
    int *ptr1=&n[0];
    int *ptr2=&i[0];
    int *ptr3=&j[0];
    printf("Enter number:");
    scanf(" %d",ptr1);
    ptr2=ptr1;
    while (*ptr1>0)
    {
        *ptr3=*ptr2%10;
        sum=sum+(*ptr3**ptr3**ptr3);
        *ptr2=*ptr2/10;
    }
    if (sum==*ptr1)
    {
        return 1;
    }
    else{
        return 0;
    }
}

File: Java Module/Assignments in c to java/Assignment-03/Question_5.java
// Check the given number is Armstrong number or not..
// Input: n = 153
// Output: Armstrong
public class Question_5 {

    public static void main(String[] args) {
            int n=153,i,j,sum=0;
    // printf("Enter number:");
    // scanf(" %d", &n);
    i=n;
    while (i>0)
    {
        j=i%10;
        sum=sum+(j*j*j);
        i=i/10;
        /* code */
    }
    if (sum==n)
    {
        System.out.printf("%d is armstrong number",n);
        /* code */
    }
    else{
        System.out.printf("%d is not armstrong numb",n);
    }
    }
}